packages/amplication-client/src/Entity/DeleteEntityField.tsx

```diff
@@ -123,3 +138,27 @@ const DELETE_ENTITY_FIELD = gql`
     }
   }
 `;
+
+const GET_ENTITY_WITH_SPECIFIC_FIELDS = gql`
+  query entity($entityId: String!, $fieldId: String!) {
+    entity(where: { id: $entityId }) {
+      id
+      name
+      displayName
+      pluralDisplayName
+      fields(where: { permanentId: { equals: $fieldId } }) {
```

Comment:
are you sure you are using the correct field for fetch? we usually use "id" and not "permanentId" which is a specific type of id that follows the field over versions